client/src/pages/Resource/index.jsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import { ResourceService, useService } from "../../services";
+
+import "./Resource.scss";
+
+export default function Resource() {
+	const [resource, setResource] = useState(undefined);
+	const resourceService = useService(ResourceService);
+	const { id } = useParams();
+
+	useEffect(() => {
+		resourceService.getOne(id).then(setResource);
+	}, [id, resourceService]);
+
+	if (!resource) {
+		return null;
+	}
+
+	return (
+		<>
+			<h2>
+				Resource &gt; <span>{resource.title}</span>
+			</h2>
+			<section>
+				<table aria-label="resource details" className="resource-details">
+					<tbody>
+						<tr>
+							<th>Title</th>
+							<td>{resource.title}</td>
+						</tr>
+						<tr>
+							<th>ID</th>
+							<td>{resource.id}</td>
+						</tr>
```

Comment:
❓ I don't see a requirement to show the id. Is this something that's been asked for elsewhere?